src/components/about/About.jsx

```diff
@@ -17,6 +18,11 @@ import { Pill } from "../common/Pill.jsx";
 import { useEffect, useState } from "react";
 import Card from "../common/card/Card.jsx";
 import { data } from "../data.js";
+import { Subtitle } from "../common/Subtitle.jsx";
+import { main } from '../../Themes.jsx'
```

Comment:
this import brings in stuff we don't need in here.
use the `withTheme` or `useTheme` hook instead.
find more info here:
https://styled-components.com/docs/advanced